lib/rpc.ts

```diff
@@ -173,9 +175,17 @@ export class Rpc extends EventEmitter {
     this._forwarders.set(fwdName, {
       name: "[FWD]" + fwdName,
       argsCheckers: null,
-      invokeImpl: (c: IMsgRpcCall) => destRpc._makeCall(c.iface, c.meth, c.args, anyChecker, fwdDest),
-      forwardMessage: (msg: IMsgCustom) => destRpc.postMessageForward(fwdDest, msg.data),
+      invokeImpl: (c: IMsgRpcCall) => destRpc._makeCall(c.iface, c.meth, c.args, anyChecker, getFwdDest(c.mdest)),
+      forwardMessage: (msg: IMsgCustom) => destRpc.postMessageForward(getFwdDest(msg.mdest), msg.data),
     });
+    function getFwdDest(mdest: string|undefined): string {
+      // when forwarding to '*' use same fwdDest
+      return (fwdName === "*") ? mdest || "" : fwdDest;
```

Comment:
I suggest we honor fwdDest if it is given, and use fwdDest of "*" to indicate "pass-through", like this:
```
public registerForwarder(fwdName: string, destRpc: Rpc, fwdDest: string = (fwdName === "*" ? "*" : ""))
   const passThru = (fwdDest === "*");
    ... _makeCall(..., anyChecker, passThru ? c.mdest : fwdDest)
```
This way the default behavior is same as you are suggesting, but fwdDest is honored if it is given, even when fwdName is "*".

Reply:
done, 
with a minor modification for handling case when `c.mdest` is undefined.